pinot-spi/src/test/java/org/apache/pinot/spi/config/table/IndexingConfigTest.java

```diff
@@ -46,6 +46,9 @@ public void testSerDe()
     indexingConfig.setOnHeapDictionaryColumns(onHeapDictionaryColumns);
     List<String> bloomFilterColumns = Arrays.asList("a", "b");
     indexingConfig.setBloomFilterColumns(bloomFilterColumns);
+    Map<String, BloomFilterConfig> bloomFilterConfigs = new HashMap<>();
+    bloomFilterConfigs.put("a", new BloomFilterConfig(0.123, 456, true));
+    indexingConfig.setBloomFilterConfigs(bloomFilterConfigs);
```

Comment:
Is this a lateral change or is it actually related to the PR?

Reply:
Lateral change unrelated to this PR. Made it because it was missing.